### ISSUE TITLE
left/right mouse click

### ISSUE THREAD

**Issue by randyschmitt** [topher-au/WoWmapper#74]
Is there a way to remap the left/right mouse clicking to another button?

**Comment by mytjake** [comment]
Yes. Please. If mouse buttons could be relocated to another button everything would be perfect.

**Comment by mytjake** [comment]
So, if you use an Xbox controller and have the Xbox Accessories app on your PC you can use the button mapping feature to assign the stick click to any other button.
